spec/fixtures/custom_plugins/kong/plugins/get-plugin-id/handler.lua

```diff
@@ -0,0 +1,10 @@
+local PluginConfigDumpHandler =  {
+  VERSION = "1.0.0",
+  PRIORITY = 1,
+}
+
+function PluginConfigDumpHandler:access(conf)
+  kong.response.exit(200, kong.plugin.get_id())
```

Comment:
Or we can write to the headers as we will add more and more `kong.plugin.get_xxx()` function into the pdk/plugin in the future?

Reply:
Not sure if that's a good idea to have a custom plugin for multiple irrelevant tests.
Maybe we could build a plugin to execute arbitrary Lua code?